package.json

```diff
@@ -41,6 +41,8 @@
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "po2json": "^1.0.0-alpha",
     "qunit": "^2.9.3",
+    "sass": "~1.32.0",
```

Comment:
Same comment as ostree: we might be able to get away with unpinning this with new PF4.

Reply:
ostree is pure PF4, and PR #187 just updated to latest PF4, so that might indeed work now. In the current PR, mini-css-extract-plugin barfs, so this needs more work anyway.